standard/expressions.md

```diff
@@ -4580,6 +4735,14 @@ The second and third operands, `x` and `y`, of the `?:` operator control the t
 - If only one of `x` and `y` has a type, and both `x` and `y` are implicitly convertible to that type, then that is the type of the conditional expression.
 - Otherwise, no expression type can be determined, and a compile-time error occurs.
 
+The run-time processing of a ref conditional expression of the form `b ? ref x : ref y` consists of the following steps:
+
+- First, `b` is evaluated, and the `bool` value of `b` is determined:
+  - If an implicit conversion from the type of `b` to `bool` exists, then this implicit conversion is performed to produce a `bool` value.
+  - Otherwise, the `operator true` defined by the type of `b` is invoked to produce a `bool` value.
```

Comment:
What if it doesn't exist (there is no such operator)?

Reply:
It’s a compile-time requirement that it does exist if there is no implicit conversion, its up on line 4717 – just further away now `ref` stuff has been added.

Might be worth considering that increased distance, but I’ll leave that decision to the talented wordsmiths among us!

Reply:
No change needed.